src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -799,6 +799,35 @@ const replaceNodeHtmlLinks = ({ wpUrl, nodeString, node }) => {
   return nodeString
 }
 
+// replaces specific string with a given string from the plugin options config
+const findAndReplaceNodeHtmlString = ({ nodeString, node, pluginOptions }) => {
+  if (pluginOptions?.findAndReplace?.length) {
```

Comment:
I think we should be looking for `pluginOptions?.searchAndReplace` according to the docs.

also, this would be a great place to check `Array.isArray(pluginOptions?.searchAndReplace)`. However, now that we have plugin options validation, duck typing checks like these won't be as important